### PR TITLE
first html task committed

### PR DESCRIPTION
**Pull request by nangchetnamaw**
pr raised as instructed.

### CODE REVIEW

File: Chetna_Mongmaw/htmltask/linked.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<meta name="author" content="chetna"/>
+<link rel="shortcut icon" type="image/x-icon" href="eco.ico" />
+</head>
+<body>
+<form action="">
+<div ">
+<h2>Greetings!!</h2>
+<br/>
+<br/>
+Name: &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp <input type="text" name="uname" /><br/><br/>
```
Comment (amandeeprikhi) [inline Chetna_Mongmaw/htmltask/linked.html]: &nbsp needs to end with a semi-colon it should be &nbsp;

**Review by amandeeprikhi — COMMENTED** [review]
Please correct the nbsp usage

File: Chetna_Mongmaw/calculator/assets/scripts/calc.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    x = x.replace(/x/g, "*")
+    x = x.replace(/÷/g, "/")
+
+
+
+    let y = eval(x) 
+    document.getElementById("result").value = y 
+}
+
+function cscreen()
```
Comment (nikhilrstg18) [inline Chetna_Mongmaw/calculator/assets/scripts/calc.js]: Indentation missing
naming convention missing
empty function not removed
function comments to brief about its purpose missing

File: Chetna_Mongmaw/calculator/assets/scripts/calc.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    x = x.replace(/sin/g, "Math.sin");
+    x = x.replace(/cos/g, "Math.cos");
+    x = x.replace(/tan/g, "Math.tan");
+    x = x.replace(/√/g, "Math.sqrt")
+    x = x.replace(/x/g, "*")
+    x = x.replace(/÷/g, "/")
+
+
+
+    let y = eval(x)
```
Comment (nikhilrstg18) [inline Chetna_Mongmaw/calculator/assets/scripts/calc.js]: get rid of eval(), find a work-around

File: Chetna_Mongmaw/calculator/calc.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./assets/styles/calc.css"/>
+    <script src="./assets/scripts/calc.js"></script>
```
Comment (nikhilrstg18) [inline Chetna_Mongmaw/calculator/calc.html]: good to see folder structure is followed

File: Chetna_Mongmaw/htmltask/linked.html (hunk 1 above)
Comment (nikhilrstg18) [reply Chetna_Mongmaw/htmltask/linked.html]: also get rid of &nbsp; this is degrading readability,
make visible adjustment using CSS syling

File: Chetna_Mongmaw/htmltask/linked.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (nikhilrstg18) [inline Chetna_Mongmaw/htmltask/linked.html:5]: missing proper indentation

File: Chetna_Mongmaw/htmltask/profilemy.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```
Comment (nikhilrstg18) [inline Chetna_Mongmaw/htmltask/profilemy.html:1]: missing proper indentation - poor readability

File: Chetna_Mongmaw/profile/assets/styles/p3.css [hunk 7]
```diff
@@ -0,0 +1,200 @@
+  /* body{
```
Comment (nikhilrstg18) [inline Chetna_Mongmaw/profile/assets/styles/p3.css]: get rid of all comments in CSS, HTML and JS files if it is not informative

File: Chetna_Mongmaw/profile/assets/styles/p3.css [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+      padding: 2rem;
+      text-align: center;
+      background-color: black;
+      margin-top: 2rem;
+      margin-bottom: 2rem;
+      color: white;
+      font-family: "Zilla Slab",sans-serif;
+
+  }
+  .body{
```
Comment (nikhilrstg18) [inline Chetna_Mongmaw/profile/assets/styles/p3.css]: is this a CSS class name?

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
Indentation missing
naming convention missing
empty function not removed
function comments to brief about its purpose missing
Missing SOLID DRY KISS in the code

File: Chetna_Mongmaw/profile/assets/styles/p3.css (hunk 8 above)
Comment (nangchetnamaw) [reply Chetna_Mongmaw/profile/assets/styles/p3.css]: corrected

File: Chetna_Mongmaw/profile/assets/styles/p3.css (hunk 7 above)
Comment (nangchetnamaw) [reply Chetna_Mongmaw/profile/assets/styles/p3.css]: unnecessary comments removed

File: Chetna_Mongmaw/htmltask/linked.html (hunk 1 above)
Comment (nangchetnamaw) [reply Chetna_Mongmaw/htmltask/linked.html]: corrected

File: Chetna_Mongmaw/htmltask/linked.html (hunk 5 above)
Comment (nangchetnamaw) [reply Chetna_Mongmaw/htmltask/linked.html:5]: corrected

**Review by nikhilrstg18 — APPROVED** [review]
Looks good to me 
Suggestion: Work on code readability improvements

**Review by amandeeprikhi — APPROVED** [review]
LGTM

**Pull request merged**